site/en/tutorials/python_quickstart.ipynb

```diff
@@ -141,7 +141,7 @@
     },
```

Comment:
<p>Can we say more than "affects the embedding"?</p><p> </p><p>e.g. something like "Separating the title from the content ensures your content is modeled correctly when calculating similarity scores." and also "To ensure reliable results, include or omit the title consistently across your corpus."</p><p> </p><p>@sidgoogle gave me a quick review on these claims :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>